File: LeetCode/Medium/1877-minimize-maximum-pair-sum-in-array/1877-minimize-maximum-pair-sum-in-array.cs
public class Solution {
    public int MinPairSum(int[] nums) {
        Array.Sort(nums);
        int l= nums.Length;
        List<int> pairs= new();
        for(int i=0;i<l/2;i++){
            pairs.Add(nums[i]+nums[l-1-i]);
        }
        return pairs.Max();
    }
}

File: LeetCode/Medium/1877-minimize-maximum-pair-sum-in-array/1877-minimize-maximum-pair-sum-in-array.py
class Solution:
    def minPairSum(self, nums: List[int]) -> int:
        nums=sorted(nums)
        m=nums[-1]
        p=[]
        p.append(m+nums[0])
        for i in range(1,len(nums)-2,2):
                p.append(nums[i]+nums[-i-1])
        return max(p)
